Library/Formula/relx.rb

```diff
@@ -0,0 +1,43 @@
+class Relx < Formula
+  homepage "https://github.com/erlware/relx"
+  url "https://github.com/erlware/relx.git", :tag => "v1.2.0"
+
+  head "https://github.com/erlware/relx.git"
+
+  depends_on "erlang"
+  depends_on "rebar"
+
+  def install
+    system "make", "escript"
+    bin.install "relx"
```

Comment:
unfortunately no, this project is very similar to rebar in that regard, the final target is a relx binary to be deployed wherever the user wishes

there are no submodules in the relx project